### PR TITLE
update the contributing and local development docs

### PR DESCRIPTION
**Pull request by clarkbw**
### Summary of Changes

Updated some of the contributing, but mostly the local development docs.

### Test Plan

- [x] Check that all the links on the current pages work / don't work
- [x] Try to test the local links

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ When a person is assigned to an issue this indicates an _intent to implement_.
  * The [integration tests](#integration-tests) will be run automatically by the CI or you can try running them locally as well
 * Once the tests have passed in the PR you must receive a review using the GitHub review system
  * To learn more about GitHub reviews take a look at their [documentation](https://help.github.com/articles/reviewing-changes-in-pull-requests/) and [video tutorial](https://youtu.be/HW0RPaJqm4g)
-* Request review from @jasonLaster or @jlongster by mentioning their names in the PR
+* We have a number of contributors reviewing PRs fairly quickly, if you feel yours has been neglected please mention  [@jasonLaster](https://github.com/jasonLaster/) by name in the PR
```
Comment (jasonLaster) [inline CONTRIBUTING.md]: how about mentioning `@debugger` to reach the debugger team.

File: docs/local-development.md [hunk 2]
```diff
@@ -113,18 +113,19 @@ index fdbdb4e..4759c14 100644
...
+The local debugger supports three themes:
 
+Light     | Dark      | Firebug
+--------- | --------- | ---------
+<a href="https://cloud.githubusercontent.com/assets/254562/20676302/4cb04a7c-b55d-11e6-855f-654395e2c26f.png"><img width="480" alt="light-theme" src="https://cloud.githubusercontent.com/assets/254562/20676302/4cb04a7c-b55d-11e6-855f-654395e2c26f.png"></a> | <a href="https://cloud.githubusercontent.com/assets/254562/20676304/4cbfbf16-b55d-11e6-9b84-3ee5595e36be.png"><img width="480" alt="dark-theme" src="https://cloud.githubusercontent.com/assets/254562/20676304/4cbfbf16-b55d-11e6-9b84-3ee5595e36be.png"></a> | <a href="https://cloud.githubusercontent.com/assets/254562/20676303/4cbb0570-b55d-11e6-98b5-d1dd124345cd.png"><img width="480" alt="firebug-theme" src="https://cloud.githubusercontent.com/assets/254562/20676303/4cbb0570-b55d-11e6-98b5-d1dd124345cd.png"></a>
```
Comment (jasonLaster) [inline docs/local-development.md]: Yes!

**Comment by jasonLaster** [comment]
Thanks! These changes really help people getting started.

File: CONTRIBUTING.md (hunk 1 above)
Comment (clarkbw) [inline CONTRIBUTING.md]: Oh right!  Good call.

**Pull request merged**